examples/README.md

```diff
@@ -5,3 +5,90 @@ This directory contains a series of examples showing how to use the Functions Fr
 A good place to start is [hello_world](hello_world). It contains a simple example, with a simple function in a single
 `.cc` file.
 
+## Create the Development Image
+
+To compile the examples you will need a Docker image with the development tools and core dependencies pre-compiled.
+To create this image run this command:
+
+```sh
+docker build -t gcf-cpp-develop -f build_scripts/Dockerfile build_scripts
+```
+
+## Creating a Docker image for the examples
+
+The examples can run in Docker containers. To compile them you first need to
+[create a development image](#create-the-development-image). Once that image is available with this image at hand you
+can create a docker image for any of the examples using:
```

Comment:
The three "image" references confuse me.

Perhaps s/with this image as hand//.

Or perhaps _"To compile them you first ... Once that image is available ... you can"_ now simplifies to _"Using the development image from above, you can"_.

Reply:
Fixed, I think.